src/js/edu-benefits/1990-rjsf/containers/ConfirmationPage.jsx

```diff
@@ -127,7 +127,7 @@ class ConfirmationPage extends React.Component {
           </ul>
         </div>
         <p>Find out what happens <a href="/education/after-you-apply">after you apply</a>.</p>
-        <p>Need help? If you have questions, call <a href="tel:888-442-4551">888-442-4551</a> (888-GI-BILL-1) from 8:00 a.m. - 7:00 p.m. ET Mon - Fri.</p>
+        <p>Need help? If you have questions, call <a href="tel:888-442-4551">1-888-442-4551</a> (888-GI-BILL-1) from 8:00 a.m. - 7:00 p.m. ET Mon - Fri.</p>
```

Comment:
We don't need to add `1` to the `888-GI-BILL-1` version?

Reply:
@jbalboni The 1 should be added to all phone numbers. Looks like you caught one where it's missing?  Can you add it?

Reply:
Updated.  Didn't add it cause it is a vanity phone number.